back/app/models/custom_form.rb

```diff
@@ -26,6 +26,9 @@ class CustomForm < ApplicationRecord
 
   delegate :project_id, to: :participation_context
 
+  before_validation :sanitize_print_start_multiloc
+  before_validation :sanitize_print_end_multiloc
```

Comment:
@jamesspeake These are the sanitization changes I made to this model, as discussed.

Not expecting you to review the whole PR, just these changes and the related tests.